src/main/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSpark.java

```diff
@@ -33,16 +33,68 @@
 import java.io.PrintStream;
 import java.util.List;
 
+/**
+ * Spark version of the first pass of the base quality score recalibration.
+ * Generates a recalibration table based on various covariates
+ * (such as read group, reported quality score, machine cycle, and nucleotide context).
+ *
+ * <p>
+ * This walker is designed to work as the first pass in a two-pass pipeline. It does a by-locus traversal operating
+ * only at sites that are not in dbSNP. We assume that all reference mismatches we see are therefore errors and indicative
+ * of poor base quality. This walker generates tables based on various user-specified covariates (such as read group,
+ * reported quality score, cycle, and context). Since there is a large amount of data one can then calculate an empirical
```

Comment:
--> This walker generates tables based on specified covariates.

Reply:
Fixed.